README.md

```diff
@@ -691,10 +695,25 @@ The following protocol specification features are not currently implemented:
   recorded in issue [100](https://github.com/yahoo/proxy-verifier/issues/100).
 * Only TCP is supported. There have been recent discussions about adding
   HTTP/3 support, which is over UDP, but work for that has not yet started.
+* the PROXY protocol support is limited to the `PROXY` command via TCP over IPv4 or IPv6. Therefore,
+  * No support for `AF_UNIX` socket address as either source and destination address.
+  * No support for UDP transport type.
+  * No support for `LOCAL` command(in v2 header).
 
 If there is no `protocol` node specified, then Proxy Verifier will default to
 establishing an HTTP/1 connection over TCP (no TLS).
 
+#### PROXY protocol support
```

Comment:
Let's re-generate the TOC at the start of the README to include this.

Also, can you add a brief description of PROXY Protocol along with a link to that unofficial RFC document? Maybe start the paragraph with something like this:

```
Generally speaking, a server sitting downstream from a proxy does not have visibility into the client's network socket information that lies behind the proxy. PROXY Protocol provides a mechanism to provide visibility for this. PROXY protocol is a network protocol that communicates a client's source and destination IP and port information via a set of bytes at the start of a TCP connection from a proxy. Here is a link to the protocol description:

https://github.com/haproxy/haproxy/blob/master/doc/proxy-protocol.txt
```

Reply:
Regenerated the ToC following the steps in the comments, which is handy.

That's a good description of the PROXY protocol. Added that in the paragraph.